tests/components/switch/test_flux.py

```diff
@@ -189,8 +189,8 @@ def test_flux_after_sunrise_before_sunset(self):
                     fire_time_changed(self.hass, test_time)
                     self.hass.pool.block_till_done()
         call = turn_on_calls[-1]
-        self.assertEqual(call.data[light.ATTR_BRIGHTNESS], 180)
-        self.assertEqual(call.data[light.ATTR_XY_COLOR], [0.431, 0.38])
+        self.assertEqual(call.data[light.ATTR_BRIGHTNESS], 119)
```

Comment:
How come these values have changed if the only thing you fixed was when update is called from a service?